lib/utils/logDeprecation.js

```diff
@@ -113,15 +112,25 @@ module.exports.flushBuffered = () => {
 
 module.exports.printSummary = () => {
   if (!bufferedDeprecations.length) return;
+
+  log.info();
+
   try {
     if (bufferedDeprecations.length === 1) {
+      log.info(style.aside("1 deprecation found: run 'serverless doctor' for more details"));
```

Comment:
This needs to be at _warning_ level. Still one issue is that Warning logs are prefixed with `WARNING:`, I've patched that at https://github.com/serverless/utils/pull/123 Once it's published we can simply write deprecation logs with `log.get("deprecation").warning("...")`